index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio header & headline section</title>
+</head>
+<body>
+    <header>
+        <a href="#home" class="active">Ahmed Adel</a>
+        <a href="#home" class="icon">
+            <i class="fa fa-bars"></i>
```

Comment:
- [ ] Add some margin and padding to the Hamburg icon to match the given design

Reply:
done